Week-3/Exercise3.2/apply.js

```diff
@@ -0,0 +1,10 @@
+let car = {
+    model: 'fortuner',
+    price: 45
+}
+
+let getCar = function(manufacturer, fuelType) {
+    console.log(`${manufacturer} manufactures ${this.model} it's on road price is ${this.price} and fuelType is ${fuelType}`)
+}
+
+getCar.apply(car, ['Toyota', 'Diesel'])
```

Comment:
Maybe you can add some explanation with real application example, for better understanding